Amplitude.xcodeproj/project.pbxproj

```diff
@@ -1264,6 +1286,10 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
```

Comment:
Is this all generated from xcode?

Reply:
Looks like it. xcode settings too. Might want to take a look at `FRAMEWORK_SEARCH_PATHS`, is this something you saw before @yuhao900914 ?

Reply:
Yes, it's auto generated by xcode. When doing the development, we need to open the Amplitude.xcworkspace instead of the whole folder. Then you will find, the AMPStorage files are gone. It cannot build the file when runing the test.